config/runtime/fstests.jinja2

```diff
@@ -1,32 +1,59 @@
 {# -*- mode: Python -*- -#}
 {# SPDX-License-Identifier: LGPL-2.1-or-later -#}
- 
+
 {%- extends 'base/python.jinja2' %}
- 
+
 {%- block python_imports %}
 {{ super() }}
 import subprocess
 {%- endblock %}
- 
-{%- block python_globals %}
-{{ super() }}
-REVISION = {{ revision }}
-{% endblock %}
- 
-{%- block python_body %}
-def check_kvm_xfstest_env():
-   try:
-       result = subprocess.run("kvm-xfstests --help", check=True, shell=True)
-       if result.returncode == 0:
-           print(f"KVM-xftests found.")
-           return True
-       else:
-           return False
-   except Exception as e:
-       print(f"Raised exception: {e}")
-       return False
- 
-def main(argv):
-   print("Checking if KVM-xfstests is available...")
-   check_kvm_xfstest_env()
+
+{% block python_job -%}
+class Job(BaseJob):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def _kernel_config(self, kdir):
+        results = {}
+        try:
+            os.chdir(kdir)
+            result = subprocess.run("kvm-xfstests install-kconfig", check=True, shell=True)
+            if result.returncode == 0:
+                if os.path.exists('.config'):
+                    print(f"Kernel config done.")
+                    fstests_result = True
+                else:
+                    print((f"Missing kernel .config file."))
+                    fstests_result = False
+            else:
+                fstests_result = False
+        except Exception as e:
+            print('Kernel config error:', e)
+            fstests_result = False
+        finally:
+            results['result'] = 'pass' if fstests_result else 'fail'
+            return results
```

Comment:
You can also just return `'pass'` or `'fail'` and use the default `submit()` method from the base class, like in `kver.jinja2`.

Reply:
now just returning pass or fail..